### ISSUE TITLE
Dicatation in Insert window - Chrome complication?

### ISSUE THREAD

**Issue by sqril** [writemonkey/wm3#340]
I just realized that there was a Writemonkey 3 and installed it today.

Yesterday, I had gotten speech recognition set up on my computer and it worked a treat with WM2.7(?).

But now with WM3 when I use speech recognition (Windows Speech Recognition in Windows 7 - not Dragon :( ) my text pops up in a little window in the upper right and I have to "Insert" it to get it into the document and then can only add to the text that has been inserted, cannot edit it.

Am I missing something?  I saw that someone using Dragon seems to be fairly successful in using dictation.

Now that I'm trying to use speech recognition to enter this into Github comment on a Chrome browser I see that it might be a browser issue and I believe the new WM is written in some sort of chrome browser format.  Correct?

I wonder if this is a Chrome setting that I can change?

I really want this to work - any suggestions?

**Comment by writemonkey** [comment]
Yes, wm is based on chromium. Dictation works well on OSX (OS built in solution), not sure for Windows though (haven't tested that yet).
i.

**Comment by sqril** [comment]
So I've tested a bunch of speech-to-text options out and I've come to the conclusion that is still fairly rough, especially in cross-platform stuff.  The only thing that seems to work at the moment is if I use the default Microsoft speech-to-text service, allow the text to come up in a bubble, hit enter, and then go back and edit via typing.

I'm using the Voicein extension in my Chrome browser to enter this and then I edit manually and it works.  But then I tested it in the WorkFlowy Chrome app and it doesn't seem to work.  There is no place to activate it.

I believe that Writemonkey is a Chromium-based program, not a Chrome app. Is that correct?

My goal with this is to use Writemonkey for a personal journal, not particularly for writing a book or anything and I wanted to use speech-to-text because I have found that it's easier for me to think my thoughts and then go back and edit them. I have a hard time getting things out of my mind in an organized fashion. 

It would be really great to have a seamless speech to text functionality via an extension or program in Writemonkey. For those of us who aren't using OSX :)

**Comment by sqril** [comment]
And I dug around but am not a programmer - I think this is describing something similar...

https://stackoverflow.com/questions/50125821/standard-way-to-build-a-chrome-extension-into-chromium/50305105

Over and out - I'm just gonna go type :)
